sdk/nodejs/src/clientUser.ts

```diff
@@ -0,0 +1,14 @@
+import { DVCPopulatedUser, DevCycleUser } from '@devcycle/js-client-sdk'
+
+export const generateClientPopulatedUser = (
+    user: DevCycleUser,
+    userAgent: string,
+): DVCPopulatedUser => {
+    return new DVCPopulatedUser(
```

Comment:
What are the other properties here? do they need to get set?

Reply:
no, they are for other kinds of initializations of the User
<img width="469" alt="image" src="https://github.com/DevCycleHQ/js-sdks/assets/1740731/1c968794-53c1-4c80-94d0-5373e5e6a0bd">